### PR TITLE
Update product name

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: akka-operator
-description: Akka Operator
+description: Akka Operator in Akka Cloud Platform
```
Comment (RayRoestenburg) [inline akka-operator/Chart.yaml:3]: Akka Cloud Platform Operator?

**Review by RayRoestenburg — APPROVED** [review]

File: akka-operator/Chart.yaml (hunk 1 above)
Comment (patriknw) [reply akka-operator/Chart.yaml:3]: could work but it's a mouthful. In docs I have used Akka Operator when referring specifically to the operator and Akka Cloud Platform for the "bigger" product picture. I'm thinking that the Akka Cloud Platform can contain more than the Akka Operator in the future.

Comment (RayRoestenburg) [reply akka-operator/Chart.yaml:3]: Would be nice to have one operator, (not have many) and of course the product can grow over time, we can always rename.

Comment (RayRoestenburg) [reply akka-operator/Chart.yaml:3]: I would have liked "Akka Cloud" as a product name but I can understand the Platform addition, to not make it sound like we are doing a full cloud service like Akka serverless...

Comment (patriknw) [reply akka-operator/Chart.yaml:3]: we are speculating, it could be other things than the operator, for example something for Telemetry

**Comment by patriknw** [comment]
Merging, and PM can request more name changes. This specific name is not much visible.

File: akka-operator/Chart.yaml (hunk 1 above)
Comment (RayRoestenburg) [reply akka-operator/Chart.yaml:3]: Oh I just meant that the name `Akka Cloud Platform Operator` would be problematic if there was more than one `operator` for Akka Cloud Platform, I thought you meant that. Anyway, no big deal in any case.

**Pull request merged**